### PR TITLE
Restyling

### PR DESCRIPTION
**Pull request by rmbielby**
# Overview

This is an overhaul of the styling to bring it more into line with other government depts and try and bring a cleaner look (see issue #21).

As well as the basic restyling, I've also added in more relevant demo content based on some LA finance data taken from the S251 Outturn publication (I've let the team lead know and coincidentally, they're in the early stages of thinking about creating a dashboard).

The plan is to provide some greater relevancy and info by expanding:
* the homepage text
* the dashboard content
* supporting extras

# Homepage text

I've only done a little of this so far, but the idea is to provide a brief intro to using the template and some examples of how to implement links. Here's the current example:

Restyling_Homepage

# Example dashboard content

The dashboard section in this new version consists of two demo tabs. Both share some top level phase and area selection inputs.

The first panel then has an example line graph of a time series:

Restyling_DashboardTab1

The second combines an additional input panel specific to this tab along with a bar chart and a table of the data:

Restyling_DashboardTab2

# Supporting extras

I've added a script that's intended to store useful bits of reference data and functions. At the moment it contains:

* GSS recommended colour scheme hex colour codes;
* Example banner, including links to separate mirrors of the site.

### CODE REVIEW

**Comment by chfoster** [comment]
second dashboard tab still spells out 'heading level 2' not h2, and should we make it so the 'app contents' link actually works and links to the dashboard pages?

**Comment by rmbielby** [comment]
> second dashboard tab still spells out 'heading level 2' not h2, and should we make it so the 'app contents' link actually works and links to the dashboard pages?

Thanks, fixed those two bits now.

File: ui.R [hunk 1]
```diff
@@ -27,179 +27,116 @@
...
-        support_links() # defined in R/supporting_links.R
-      ),
-      # Footer ====================================================================
-
-      shinyGovstyle::footer(TRUE)
-    ) # End of navBarPage
+    shinyGovstyle::header(
+      main_text = "DfE",
+      secondary_text = "DfE Shiny Template",
+      logo = "images/DfE_logo.png"
```
Comment (cjrace) [inline ui.R:74]: Can we make the DfE text be an actual link to the DfE page on gov.uk?

**Review by cjrace — CHANGES_REQUESTED** [review]
Fairly big issue with the top banner and footer not scaling with the main content. I had a quick play and there seems to be some joy in moving the `govlayout()` function around, though I've not got it working better yet. Examples of the issue

image

image

image

**Comment by rmbielby** [comment]
> Fairly big issue with the top banner and footer not scaling with the main content. I had a quick play and there seems to be some joy in moving the `govlayout()` function around, though I've not got it working better yet. Examples of the issue
> 
> image
> 
> image
> 
> image

Now fixed by switching the use of fluidPage() and fluidRow() in the dashboard panels to gov_main_layout() and gov_row() respectively. The header still behaves the same way, but the content below it now scales along with it.

File: ui.R [hunk 2]
```diff
@@ -27,179 +27,116 @@
...
+    ),
+    shinyGovstyle::banner(
+      "beta banner",
+      "beta",
+      paste0(
+        "This Dashboard is in beta phase and we are still reviewing performance and reliability. ",
+        "In case of slowdown or connection issues due to high demand, we have produced two instances of this site which can be accessed at the following links: ",
+        "<a href=", site_primary, ">Primary</a> and ",
+        "<a href=", site_overflow, ">Mirror</a>."
```
[the remark this answers is not in the document]
Comment (rmbielby) [reply ui.R]: Updated to Site 1/Site 2. Links are already tracked just with the basic GA I think, without the need for custom events. I've added a link ID to each, which I think should come up in GAs.

File: ui.R (hunk 1 above)
Comment (rmbielby) [reply ui.R:75]: Good point. Done now.

**Pull request merged**